libraries/src/MVC/Controller/ApiController.php

```diff
@@ -554,4 +565,28 @@ protected function preprocessSaveData(array $data): array
 	{
 		return $data;
 	}
+
+	/**
+	 * Method to set request filter parameters in model state.
+	 *
+	 * @param   array  $data  An array of input data.
+	 *
+	 * @return  void
+	 */
+	protected function setModelFilterStateFromRequest()
+	{
+		$apiFilterInfo = $this->input->get('filter', [], 'array');
+		$filter        = InputFilter::getInstance();
+
+		foreach ($this->modelFilterStateMap as $apiFilterName => $modelState)
+		{
+			if (!array_key_exists($apiFilterName, $apiFilterInfo) || trim($apiFilterInfo[$apiFilterName]) !== '')
+			{
+				continue;
+			}
+
+			$this->modelState->set($modelState['name'],
+				$filter->clean($apiFilterInfo[$apiFilterName], $modelState['type']));
```

Comment:
```suggestion
			$this->modelState->set(
				$modelState['name'],
				$filter->clean($apiFilterInfo[$apiFilterName], $modelState['type'])
			);
```
Fix to make PHP code style check pass in drone.

Reply:
@richard67 How do I know when or what to distribute over multiple lines? The [Joomla! Coding Standards](https://developer.joomla.org/coding-standards/php-code.html) does not mention anything about proper formatting function calls. What makes it worse, is that the [Basic Guidelines](https://developer.joomla.org/coding-standards/basic-guidelines.html) state the following:

> There is no maximum limit for line lengths in files, however, a notional value of about 150 characters is recommended to achieve a good level of readability without horizontal scrolling. Longer lines are permitted if the nature of the code for individual lines requires it and line breaks would have an adverse affect on the final output (such as for mixed PHP/HTML layout files).

Are there guidelines on how to please the drone? :D

Reply:
> Are there guidelines on how to please the drone? :D

@pjdevries Yes: The the log file from drone. It can be found when following the "Details" right beside the drone step at the bottom of the PR (might need to use "show all checks" first to expand them) and then in the list on the left hand side select the failed check, in this case it was PHPCS to see the log file at the right side.

Reply:
@richard67 Thanx for the feed back. Not the answer I was hoping for though :) I would prefer to provide drone acceptable code right away. Is there no documentation for the code style the drone expects?